provider/README.md

```diff
@@ -75,6 +75,31 @@ Response bodies must be a `UTF-8` encoded JSON object and must minimally include
 }
 ```
 
+#### Provider HTTP Codes
+
+* **200:** OK: operation successful.
+* **400:** Bad request.
+* **401:** Unauthorized: Invalid, expired, or insufficient scope of token.
+* **404:** Not Found: Object(s) do not exist.
+* **500:** Internal server error.
+
+#### Error Format
+
+```json
+{
+    "version": "x.y.z",
```

Comment:
I have a feeling that including `version` in the error response might be more confusing than it is worth. E.g. is that the version that was requested? Or the version that the Provider would have responded with had the request been successful? Or...?

Reply:
If I take it out, I'll have to modify agency too (which is fine).